src/plugins/presentation_util/public/components/floating_actions/floating_actions.scss

```diff
@@ -13,7 +13,7 @@
     z-index: $euiZLevel9; // the highest possible z-level
   }
 
-  &:hover, &:focus-within {
+  &:hover, &:focus {
```

Comment:
This actually doesn't work as a fix - the reason I chose `focus-within` as a target here is because we want the hover actions to show up as long as at least one of the elements inside the `presentationUtil__floatingActionsWrapper` has focus, like so:

![Jun-13-2023 12-54-43](https://github.com/elastic/kibana/assets/8698078/d0efcf0e-eaef-4d77-b06d-240ffa337868)

By switching to targetting only `focus`, this no longer happens:

![Jun-13-2023 12-52-20](https://github.com/elastic/kibana/assets/8698078/91b427e5-2b0b-4cbf-896a-1b55ce3cd083)